patient-portal-backend/src/main/java/com/vaadin/demo/service/DBInitService.java

```diff
@@ -91,7 +91,7 @@ private void createPatients() {
                 JournalEntry journalEntry = new JournalEntry();
                 journalEntry.setDate(cal.getTime());
                 journalEntry.setAppointmentType(AppointmentType.values()[random.nextInt(AppointmentType.values().length)]);
-                journalEntry.setEntry(lorem.getParagraphs(1, 4));
+                journalEntry.setEntry("lorem.getParagraphs(1, 4)");
```

Comment:
Ahh, EclipseLink isn't picking up the @Size(max = 10000) annotation for this column and defaulted to creating the journal entry table with this column with a max size of 255 characters. In order to save time, It was commented out- I'll look into it now.